openslides/agenda/models.py

```diff
@@ -281,15 +281,16 @@ def calc_item_no(self):
 
     def sibling_no(self):
         """
-        Counts how many AGENDA_ITEMS with the same parent (siblings) have a
+        Counts how many agenda items with the same parent (siblings) have a
         smaller weight then this item.
 
-        Returns this number + 1 or 0 when self is not an AGENDA_ITEM.
+        Returns this number + 1.
         """
-        return Item.objects.filter(
-            parent=self.parent,
-            type=self.AGENDA_ITEM,
-            weight__lte=self.weight).count()
+        result = 0
+        for item in Item.objects.filter(parent=self.parent, weight__lte=self.weight):
+            if not item.is_hidden():
+                result += 1
+        return result
```

Comment:
Then you should fix the count query instead of fetching all data from the db and counting in python.

You problably have to add .exclude(type=self.HIDDEN_ITEM)